src/convertToObject.js

```diff
@@ -5,8 +5,21 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = sourceString
+    .split(';')
+    .filter((line) => line.trim() !== '')
+    .reduce((object, el) => {
+      let [key, value] = el.split(':').map((item) => item.trim());
+
+      key = key.split('').join('').trim();
+      value = value.split('').join('').trim();
+
+      return Object.assign(object, { [key]: value });
+    }, {});
+
+  return obj;
```

Comment:
[CODE STYLE] - Avoid variable namings like `obj`, `object`. It will decrease code readability. Consider using more descriptive names that reflect the content of the variable or its purpose, such as `styleObject` or `styleMap`.